server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
Conversely, doing it in the server makes it more likely to accidentally change due to e.g. a configuration problem.

@mhammond you know what would successfully detect such bustage in a timely manner?  TPS...